app/src/test/webapp/app/error/error.ctrl.test.js

```diff
@@ -178,4 +193,68 @@ describe("ErrorCtrl", function() {
         });
     });
   });
+
+  describe("success after calling resubmit", function() {
+    it("should display a success message when the message is resubmitted", function() {
+      spyOn(messageCenterService, 'add');
+
+      $controller("ErrorCtrl", {
+        $scope: $scope
+      });
+
+      $scope.message = resubmittableMessage;
+
+      $scope.resubmitMessage();
+      $scope.$apply();
+
+      expect(messageCenterService.add).toHaveBeenCalledWith('success', 'Resubmit successful!', { "timeout": 3000 });
+    });
+  });
+
+  describe("failure after calling resubmit", function() {
+    beforeEach(function resubmitWithServerProblem() {
```

Comment:
function name is a little misleading; calling it does not resubmit with a server problem, it causes any resubmits to fail